lib/strategy.js

```diff
@@ -188,6 +190,8 @@ AcpStrategy.prototype.authenticate = function(req, options = {}) {
 						null,
 						function (err, redirectBody, res) {
 							if (err || !(res.statusCode >= 200 && res.statusCode <= 299)) {
+								console.log('Redirect request to ', res.headers.location, ', failed ', res.statusCode);
```

Comment:
same safe get comment from above.
also, just noticed the function on L191 receives `res` as the 3rd param, but `res` also exists in the outer scope, like on L181. just wanna make sure we're looking at the proper `res` object

Reply:
Yeah i will add the saftey, this code was all copied from the original package, which is why i dont think it was there to begin with